ggplot/scales/__init__.py

```diff
@@ -1,30 +1,122 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from .scale_colour_brewer import scale_colour_brewer
-from .scale_colour_brewer import scale_colour_brewer as scale_color_brewer
-from .scale_colour_gradient import scale_colour_gradient
-from .scale_colour_gradient import scale_colour_gradient as scale_color_gradient
-from .scale_colour_gradient import scale_colour_gradient as scale_colour_gradient2
-from .scale_colour_gradient import scale_colour_gradient as scale_color_gradient2
-from .scale_colour_manual import scale_colour_manual
-from .scale_colour_manual import scale_colour_manual as scale_color_manual
-from .scale_facet import scale_facet_grid
-from .scale_facet import scale_facet_wrap
-from .scale_x_continuous import scale_x_continuous
-from .scale_x_discrete import scale_x_discrete
-from .scale_x_date import scale_x_date
-from .scale_y_continuous import scale_y_continuous
-from .scale_y_discrete import scale_y_discrete
-from .scale_reverse import scale_y_reverse, scale_x_reverse
-from .scale_log import scale_x_log, scale_y_log
-from .scale_log import scale_x_log as scale_x_log10
-from .scale_log import scale_y_log as scale_y_log10
 
-__all__ = ['scale_color_brewer', 'scale_colour_brewer',
+# color
+from .scale_color import scale_color_brewer, scale_colour_brewer
```

Comment:
Somehow this long list screams for a way to not duplicate the list. Probably like adding doing a `{<from>:['items',...],...}` and then do some [import magic](https://docs.python.org/2/library/functions.html#__import__) to load the thing and put it in `__all__` in one small nested loop...